homeassistant/components/plant/const.py

```diff
@@ -0,0 +1,42 @@
+"""Const for Plant."""
+from typing import Final
+
+from homeassistant.const import ATTR_TEMPERATURE
+
+DOMAIN: Final = "plant"
+
+PLANT_NAME = "plant_name"
+SENSORS = "sensors"
+
+MOISTURE_SENSOR = "moisture"
+BATTERY_SENSOR = "battery"
+TEMPERATURE_SENSOR = ATTR_TEMPERATURE
```

Comment:
I think this was bad behavior in the previous implementation. It should be just the string to avoid confusion:
```suggestion
TEMPERATURE_SENSOR = "temperature"
```